spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/PubSubAcknowledger.java

```diff
@@ -22,9 +22,18 @@
  * Acknowledges Pub/Sub messages based on their subscription name and ack ID.
  *
  * @author João André Martins
+ * @author Doug Hoard
  */
 public interface PubSubAcknowledger {
 
+	/**
+	 * (N)acknowledgement of messages will be asynchronous
+	 * @param asyncAck
+	 *
+	 * @since 1.1
+	 */
+	void setAsyncAck(boolean asyncAck);
```

Comment:
I don't this that is going to be as a general contract of the `PubSubAcknowledger`.
Some custom implementation might be just far away from the standard Pub/Sub.

This is an API of the `DefaultSubscriberFactory` indeed, but it is really fully tied with the `DefaultPubSubAcknowledger`.

Reply:
Fixed - removed from the constract of `PubSubAcknowledger`